cmd/kubeadm/app/cmd/token.go

```diff
@@ -53,25 +53,24 @@ func NewCmdToken(out io.Writer, errW io.Writer) *cobra.Command {
 	var dryRun bool
 	tokenCmd := &cobra.Command{
 		Use:   "token",
-		Short: "Manage bootstrap tokens.",
+		Short: "Manage Bootstrap Tokens.",
 		Long: dedent.Dedent(`
-			This command will manage Bootstrap Token for you.
-			  Please note this usage of this command is optional, and mostly for advanced users.
+			This command will manage a Bootstrap Token for you.
```

Comment:
Maybe just "This command manages bootstrap tokens. It is optional and needed only for advanced use cases."?